tests/Element/TagInputTest.php

```
@@ -671,24 +786,40 @@ public function testRenderWithTitle(): void
+    public function testRenderWithToString(): void
     {
         self::assertEquals(
             <<<HTML
-            <input translate="no">
+            <input>
             HTML,
             LineEndingNormalizer::normalize(
-                TagInput::tag()->translate('no')->id(null)->render(),
+                (string) TagInput::tag()
+                    ->id(null),
             ),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\PHPForge\Support\LineEndingNormalizer' in method 'testRenderWithToString'. (undefined)

(StaticAccess)